kubernetes/chart/zulip/values.yaml

```diff
@@ -139,6 +139,26 @@ zulip:
     SETTING_EMAIL_USE_SSL: "False"
     SETTING_EMAIL_USE_TLS: "True"
     ZULIP_AUTH_BACKENDS: "EmailAuthBackend"
+  # -- Mount environment variables from secrets. Use the secret name.
+  # If you have a secret with env. variables created named zulip-secrets
+  # you will add:
+  # ```
+  # envSecrets:
+  #   - zulip-secrets
+  #   - other-secrets
+  # The secrets file you manually create in the namespace, can look something
+  # like this (secrets need to be base64 encoded):
+  # ---
+  # apiVersion: v1
+  # kind: Secret
+  # metadata:
+  #   name: zulip-secrets
+  # type: Opaque
+  # data:
+  #   SECRETS_email_password: MTIzNDU2Nzg5
+  #   SECRET: Zm9vaXNiYXI=
+  envSecrets:
+    []
```

Comment:
Is it useful/necessary to have the `[]` here?

Reply:
You mean the newline?

There is a mix of practice of this.

This:

```yaml
  envSecrets:
    []
```

And this:

```yaml
  envSecrets: []
```

Have the same meaning, but I like the first one better - because it also indicates that the values that it is entered should be on a new line:

```yaml
  envSecrets:
    - secret_ONE: ERTY
    - secret_TWO: YTRE
```    

So I think it helps the user. 

But as there are mix of standards in the helm chart, like:

```yaml
podSecurityContext:
  {}
```

And 
```yaml
podAnnotations: {}
```

There should be maybe a consensus around what should be used. I like the newline myself, as I think it helps to guide the user.